benchmarks/src/util/options.rs

```diff
@@ -37,11 +37,6 @@ pub struct CommonOpt {
     /// Activate debug mode to see more details
     #[structopt(short, long)]
     pub debug: bool,
-
-    /// If true, will use StringView/BinaryViewArray instead of String/BinaryArray
-    /// when reading ParquetFiles
-    #[structopt(long)]
-    pub force_view_types: bool,
```

Comment:
Should we keep this (or a differently-named) flag as a kill-switch?

Reply:
👍 There is a kill switch (in the description of this PR)

> > set datafusion.execution.parquet.schema_force_view_types = false;
> 0 row(s) fetched.
> Elapsed 0.000 seconds.


This particular code is for the benchmark drivers and I don't think it is super valuable to retain the benchmark in both configurations